ang/civicase/activity/services/activity-feed-measurements.service.js

```diff
@@ -17,5 +17,39 @@
 
       return topOffset;
     }
+
+    /**
+     * Returns the collective height of all contact tab children
+     *
+     * @return {Number}
+     */
+    function civicrmContactTabHeight () {
+      var height = 0;
+      var $civicrmContactTabChildren = $('.crm-contact-tabs-list').children();
+
+      _.each($civicrmContactTabChildren, function (child) {
+        height += $(child).height();
+      });
+
+      return height;
+    }
+
+    /**
+     * Set height of the sent element for scrolling
```

Comment:
of the *given* element

Reply:
Can you explain more about the function in the description? Why does it care about `$civicrmContactTabs.length` and gives different heights?